doc/source/ray-air/examples/serving_guide.ipynb

```diff
@@ -0,0 +1,923 @@
+      "\u001b[2m\u001b[36m(HTTPProxyActor pid=60984)\u001b[0m INFO:     Started server process [60984]\n",
+      "\u001b[2m\u001b[36m(ServeController pid=60981)\u001b[0m INFO 2022-06-02 19:31:55,191 controller 60981 deployment_state.py:1221 - Adding 1 replicas to deployment 'XGBoostService'.\n"
+     ]
+    }
+   ],
+   "source": [
+    "from ray.ml.predictors.integrations.xgboost import XGBoostPredictor\n",
+    "from ray import serve\n",
+    "from ray.serve.model_wrappers import ModelWrapperDeployment\n",
+    "from ray.serve.http_adapters import pandas_read_json\n",
+    "\n",
+    "\n",
+    "serve.start(detached=True)\n",
+    "deployment = ModelWrapperDeployment.options(name=\"XGBoostService\")\n",
+    "\n",
+    "deployment.deploy(\n",
+    "    XGBoostPredictor, result.checkpoint, http_adapter=pandas_read_json\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's send a request through HTTP."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[{'predictions': 0.1142289936542511}]\n"
+     ]
+    },
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "\u001b[2m\u001b[36m(HTTPProxyActor pid=60984)\u001b[0m INFO 2022-06-02 19:32:00,604 http_proxy 127.0.0.1 http_proxy.py:320 - POST /XGBoostService 307 5.4ms\n",
+      "\u001b[2m\u001b[36m(XGBoostService pid=60988)\u001b[0m INFO 2022-06-02 19:32:00,603 XGBoostService XGBoostService#LOYoUm replica.py:484 - HANDLE __call__ OK 0.3ms\n",
+      "\u001b[2m\u001b[36m(HTTPProxyActor pid=60984)\u001b[0m INFO 2022-06-02 19:32:00,658 http_proxy 127.0.0.1 http_proxy.py:320 - POST /XGBoostService 200 49.8ms\n",
+      "\u001b[2m\u001b[36m(XGBoostService pid=60988)\u001b[0m INFO 2022-06-02 19:32:00,656 XGBoostService XGBoostService#LOYoUm replica.py:484 - HANDLE __call__ OK 46.8ms\n"
+     ]
+    }
+   ],
+   "source": [
+    "import requests\n",
+    "\n",
+    "sample_input = test_dataset.take(1)\n",
+    "sample_input = dict(sample_input[0])\n",
+    "\n",
+    "output = requests.post(deployment.url, json=[sample_input]).json()\n",
+    "print(output)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "It works! As you can see, `ModelWrapper` is one of the core component in Ray AIR that deploy trained checkpoint as live endpoint. You can find more end-to-end examples with your specific frameworks in the [examples](air-examples-ref) page.\n",
+    "\n",
+    "This tutorial is gear towards in depth understanding of ModelWrappers, in particular, we'll demonstrate:\n",
+    "- How to serve a predictor accepting array input.\n",
```

Comment:
This section is essentially a table of contents. Could we add links to the corresponding sections (if that's possible in Jupyter Notebooks) and numbering that corresponds to their sections? It might be worth making a separate headline just for this part called "Table of Contents".